cluster/v1beta1/helpers_test.go

```diff
@@ -388,201 +388,320 @@ func TestGetClusterSetsOfCluster(t *testing.T) {
 	}
 }
 
-type placementDecisionGetter struct {
-	client client.Client
+type FakePlacementDecisionGetter struct {
+	FakeDecisions []*PlacementDecision
```

Comment:
What's the purpose for this being an array of pointers? It seems it'd be much more straightforward for the users to not have the pointer there, but maybe I'm missing an implementation detail.

Reply:
This is the testing code to provide a fake placementDecisionGetter to easily initiate some decisions. 
In the real code, the PlacementDecisionGetter would be like this https://github.com/open-cluster-management-io/ocm/blob/main/pkg/work/helper/helpers.go#L475

Reply:
@dhaiducek ^^^^

Reply:
The placementDecisionGetter needs to be implemented by developer self, needs to implement List() interface. 

In the example of https://github.com/open-cluster-management-io/ocm/blob/main/pkg/work/helper/helpers.go#L475, the clusterlister is passed from https://github.com/open-cluster-management-io/ocm/blob/main/pkg/work/hub/controllers/manifestworkreplicasetcontroller/manifestworkreplicaset_controller.go#L120 `placementInformer.Lister()`.